subprojects/build-cache-packaging/src/test/groovy/org/gradle/caching/internal/packaging/impl/TarBuildCacheEntryPackerPermissionTest.groovy

```diff
@@ -59,6 +59,13 @@ class TarBuildCacheEntryPackerPermissionTest extends AbstractTarBuildCacheEntryP
         0 * _
 
         where:
-        mode << [ "0644", "0755" ]
+        mode << [
```

Comment:
How does this test do anything related to permissions on Windows? The interesting parts (like the actual changing of access) is mocked out?

Reply:
Ah...